test/ruleguard/dup.go

```diff
@@ -0,0 +1,22 @@
+// go:build ruleguard
+package gorules
```

Comment:
let's have the package's name same as folder's name, it confuses a bit

Reply:
done

Reply:
thanks